Documentation/getting-started.md

```diff
@@ -20,18 +20,22 @@ Getting started
 git clone https://github.com/stratisproject/StratisBitcoinFullNode.git  
 ```
 
-3. Checkout the develop branch (or master without latest changes)
+3. Checkout the master branch
 ```
 cd StratisBitcoinFullNode
-git checkout develop
+git checkout master
 ```
 
 4. Restore the project packages  
 ```
 dotnet restore
 ```
+5. Go into the Stratis.BitcoinD folder
```

Comment:
was making the cd its own step necessary?  Seems like just leaving it as part of the existing step 5 was sufficient, IMHO?